Dockerfile

```
@@ -9,9 +9,9 @@ RUN ./config/autorun.sh
 RUN ./configure --prefix=/arcus --enable-zk-integration
 RUN make && make install
 
-FROM centos:7 AS base
+FROM rockylinux:8 AS base
```

```suggestion
FROM rockylinux:8-minimal AS base
```

`minimal` image 사용하면 어떤가요?

---

minimal에 yum이 없습니다.